### PR TITLE
cloud: ovirt: Add ovirt_disk_facts module

### PR DESCRIPTION
**Pull request by KKoukiou**
##### SUMMARY
Add ovirt_disks_facts module

##### ISSUE TYPE
 - New Module Pull Request

##### COMPONENT NAME
ovirt

##### ANSIBLE VERSION
devel

##### ADDITIONAL INFORMATION
Nothing special done here, it's mostly copy paste from ovirt_vms_facts.py

### CODE REVIEW

File: lib/ansible/modules/cloud/ovirt/ovirt_disks_facts.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: ovirt_disks_facts
+short_description: Retrieve facts about one or more oVirt/RHV virtual machines
```
Comment (machacekondra) [inline lib/ansible/modules/cloud/ovirt/ovirt_disks_facts.py]: s/virtual machines/disks

File: lib/ansible/modules/cloud/ovirt/ovirt_disks_facts.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016 Red Hat, Inc.
```
Comment (machacekondra) [inline lib/ansible/modules/cloud/ovirt/ovirt_disks_facts.py]: 2017

File: lib/ansible/modules/cloud/ovirt/ovirt_disks_facts.py [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+description:
+    - "Retrieve facts about one or more oVirt/RHV Disks."
+notes:
+    - "This module creates a new top-level C(ovirt_disks) fact, which
+       contains a list of disks."
+options:
+    pattern:
+      description:
+        - "Search term which is accepted by oVirt/RHV search backend."
+        - "For example to search VM X from cluster Y use following pattern:
```
Comment (machacekondra) [inline lib/ansible/modules/cloud/ovirt/ovirt_disks_facts.py]: s/VM/disk/
s/cluster/storage

File: lib/ansible/modules/cloud/ovirt/ovirt_disks_facts.py [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+           name=X and cluster=Y"
```
Comment (machacekondra) [inline lib/ansible/modules/cloud/ovirt/ovirt_disks_facts.py]: name=X and Storage.name=Y

**Review by machacekondra — COMMENTED** [review]
Looks good, just few minor comments. Thanks a lot.

**Comment by machacekondra** [comment]
@mwperina

File: lib/ansible/modules/cloud/ovirt/ovirt_disks_facts.py (hunk 4 above)
Comment (KKoukiou) [reply lib/ansible/modules/cloud/ovirt/ovirt_disks_facts.py]: Done

File: lib/ansible/modules/cloud/ovirt/ovirt_disks_facts.py (hunk 3 above)
Comment (KKoukiou) [reply lib/ansible/modules/cloud/ovirt/ovirt_disks_facts.py]: Done

File: lib/ansible/modules/cloud/ovirt/ovirt_disks_facts.py (hunk 1 above)
Comment (KKoukiou) [reply lib/ansible/modules/cloud/ovirt/ovirt_disks_facts.py]: Done

File: lib/ansible/modules/cloud/ovirt/ovirt_disks_facts.py (hunk 2 above)
Comment (KKoukiou) [reply lib/ansible/modules/cloud/ovirt/ovirt_disks_facts.py]: Done

**Review by machacekondra — APPROVED** [review]

**Comment by machacekondra** [comment]
shipit

**Comment by machacekondra** [comment]
@ryansb

**Review by mwperina — APPROVED** [review]
👍

**Comment by KKoukiou** [comment]
@ryansb @machacekondra ping

**Comment by mwperina** [comment]
shipit

**Comment by machacekondra** [comment]
bot_status

**Comment by samdoran** [comment]
@KKoukiou Can you please rename this `ovirt_disk_facts`? See [this proposal](https://github.com/ansible/proposals/issues/10) for more details. I'll merge it after the name is changed. Thanks!

**Comment by KKoukiou** [comment]
@samdoran Done.

**Review by mwperina — APPROVED** [review]
shipit

**Comment by samdoran** [comment]
Merged to `devel` for inclusion in 2.5. Thank you!

**Pull request merged**